packages/vm/src/bloom/index.ts

```diff
@@ -67,10 +67,8 @@ export class Bloom {
    * Bitwise or blooms together.
    */
   or(bloom: Bloom) {
-    if (isTruthy(bloom)) {
```

Comment:
Hmm, I'm not so sure why this check was even here..? How come this would be "falsy"? This should not be possible right?

Reply:
Indeed, I think some of those unnecessary checks might have been ported over from the pre-TS days